pal/src/map/common.cpp

```diff
@@ -49,7 +49,7 @@ INT W32toUnixAccessControl( IN DWORD flProtect )
         MemAccessControl = PROT_EXEC | PROT_READ | PROT_WRITE;
         break;
     case PAGE_EXECUTE :
-        MemAccessControl = PROT_EXEC;
+        MemAccessControl = PROT_EXEC | PROT_READ; // WinAPI PAGE_EXECUTE also implies readable
```

Comment:
When `PAGE_EXECUTE` is set, for example not sure `ReadProcessMemory` could read that area? however, if we set `PAGE_EXECUTE_READ` that would be another story. There is no such thing on Linux and we mimic these tags to `PROT_*` And so, if we set `PROT_READ` we make that area truly readable. Do we really want that?